### PR TITLE
[DOC] HPO documentation

### PR DESCRIPTION
**Pull request by eunwoosh**
PR for HPO documentation

### CODE REVIEW

**Review by jaegukhyun — COMMENTED** [review]
Overall explanation for HPO contents look good to me, however I prefer to explain overall hpo configure steps after using otx build. What do you think?

**Comment by sungmanc** [comment]
> Overall explanation for HPO contents look good to me, however I prefer to explain overall hpo configure steps after using otx build. What do you think?

Agree with this opinion, however, we can add the detailed explanation in another PR. So, I prefer to merge like as is

**Review by sungmanc — DISMISSED** [review]

**Comment by eunwoosh** [comment]
Thanks for comments @jaegukhyun , @sungmanc . I added that step in docs. please check that :)

File: docs/source/guide/explanation/additional_features/hpo.rst [hunk 1]
```diff
@@ -1,4 +1,184 @@
...
+Key features of OpenVINO™ Training Extensions include:
+
+- **Easy usability** : By using time as the control parameter, OpenVINO™ Training Extensions offers a straightforward and intuitive interface for users.
+
+- **Auto-config** : The automatic configuration feature sets internal control parameters automatically, ensuring that HPO finishes within the given time constraint.
+
+- **Scalability** : OpenVINO™ Training Extensions offers both sequential and parallel methods, making it scalable for different training environments. If you have multiple GPUs, you can take advantage of all available GPU resources to accelerate HPO.
+
+**************
+Simpe HPO Exapmle
```
Comment (kprokofi) [inline docs/source/guide/explanation/additional_features/hpo.rst]: It should be located in tutorials/advanced/
The explanation section is only for describing algorithm

1) Please, add description about what algorithm is used for HPO with link to articles and briefly how it works
2) Add overview of the HPO config and what parameters can be optimized by HPO and how to set them (how to set time limits, lower and higher boundaries for parameters, etc.)
3) Add just one line general example how to enable it here

Everything else should be moved to tutorials

**Review by kprokofi — CHANGES_REQUESTED** [review]

File: docs/source/guide/explanation/additional_features/hpo.rst (hunk 1 above)
Comment (eunwoosh) [reply docs/source/guide/explanation/additional_features/hpo.rst]: I applied your comment. Could you check it?

**Review by kprokofi — APPROVED** [review]
Thank you very much! Such a deep explanation section for HPO and good tutorial as well

**Review by harimkang — APPROVED** [review]

**Pull request merged**